kpi/tests/test_mock_data_exports.py

```diff
@@ -678,33 +678,10 @@ def test_anotheruser_can_export_when_submissions_publicly_shared(self):
 
         anonymous_user = get_anonymous_user()
 
-        assert self.asset.has_perm(self.anotheruser, PERM_VIEW_ASSET) == False
-        assert PERM_VIEW_ASSET not in self.asset.get_perms(self.anotheruser)
-        assert self.asset.has_perm(self.anotheruser, PERM_CHANGE_ASSET) == False
-        assert PERM_CHANGE_ASSET not in self.asset.get_perms(self.anotheruser)
-
+        assert not self.asset.has_perm(self.anotheruser, PERM_VIEW_SUBMISSIONS)
         # required to export
-        self.asset.assign_perm(self.anotheruser, PERM_CHANGE_ASSET)
-
-        assert self.asset.has_perm(self.anotheruser, PERM_VIEW_ASSET) == True
-        assert PERM_VIEW_ASSET in self.asset.get_perms(self.anotheruser)
-        assert self.asset.has_perm(self.anotheruser, PERM_CHANGE_ASSET) == True
-        assert PERM_CHANGE_ASSET in self.asset.get_perms(self.anotheruser)
-
-        assert (
-            self.asset.has_perm(self.anotheruser, PERM_VIEW_SUBMISSIONS)
-            == False
-        )
-        assert PERM_VIEW_SUBMISSIONS not in self.asset.get_perms(
-            self.anotheruser
-        )
-
         self.asset.assign_perm(anonymous_user, PERM_VIEW_SUBMISSIONS)
-
-        assert (
-            self.asset.has_perm(self.anotheruser, PERM_VIEW_SUBMISSIONS) == True
-        )
-        assert PERM_VIEW_SUBMISSIONS in self.asset.get_perms(self.anotheruser)
+        assert self.asset.has_perm(self.anotheruser, PERM_VIEW_SUBMISSIONS)
```

Comment:
i can't add a comment on the actual lines because they haven't been changed by this PR, but my comments on the v2 method about issue reference format (`#1234` or `org/repo#1234`) and unnecessary manual resetting of the database state apply to this method as well